.circleci/config.yml

```diff
@@ -0,0 +1,47 @@
+version: 2
+jobs:
+  build:
+    working_directory: /temp
+    docker:  
+      - image: mono:latest
+    environment:
+      DOTNET_SKIP_FIRST_TIME_EXPERIENCE: 1
+      DOTNET_CLI_TELEMETRY_OUTPUT: 1
+    steps:
+      - checkout
+      - run: msbuild Analytics/Analytics.csproj /t:restore
```

Comment:
Could we make all these `run` steps as Makefile targets (e.g. https://github.com/segmentio/analytics-ios/blob/dev/circle.yml#L20-L23).

This makes it easy to run the same commands as we do on CI locally as well.